store/index.js

```diff
@@ -299,11 +295,13 @@ export const getters = {
             // B. Save a list of missing values for this column if,
             // 1) the column has an entry in the data dictionary and,
             // 2) if a missing values list for the column has already been made
-            if ( column in p_state.dataDictionary.annotated &&
-                "missingValues" in p_state.dataDictionary.annotated[column] ) {
+
+            // NOTE: commenting out condition checks because all columns should be in data dictionary and all have missing values lists
+            // if ( column in p_state.dataDictionary.annotated &&
+            //    "missingValues" in p_state.dataDictionary.annotated[column] ) {
```

Comment:
Is there a reason why you kept this commented code?

Reply:
Nope. Just missed this. I'll clean this up too.